packages/vuetify/src/components/VSelect/VSelect.tsx

```diff
@@ -85,9 +86,12 @@ export const makeSelectProps = propsFactory({
     default: '$vuetify.noDataText',
   },
   openOnClear: Boolean,
+  readonly: Boolean,
```

Comment:
Is this related?

Reply:
No. It is a relic of a commit I have removed, since it was unrelated to the issue at hand: https://github.com/vuetifyjs/vuetify/commit/6d6c1dc43eb3d9018420ebdbee57ee2b8ff51e42

I will remove it.